Finjector.Web/ClientApp/src/pages/Landing.test.tsx

```diff
@@ -41,14 +29,11 @@ describe("Landing", () => {
   });
 
   it("loads saved charts", async () => {
-    // setup mock
-    mockSavedChartsCall();
-
     // render component
     render(wrappedView());
 
     // should see list of saved charts (3 from our mock data)
-    await waitFor(() => {
+    await vi.waitFor(() => {
```

Comment:
this is interesting because I'm able to do regular `await waitFor(() => {` just fine in my vite projects.  Missing an include in the tsconfig.json probably